index.html

```diff
@@ -418,6 +419,58 @@ <h2>Transferable MediaStreamTrack</h2>
       <p>The underlying source is supposed to be kept alive between the transfer and transfer-receiving steps, or as long as the data holder is alive.
       In a sense, between these steps, the data holder is attached to the underlying source as if it was a track.</p>
     </div>
+    <h3>MediaStreamTrack frame counters</h3>
+    <div>
+      <p>
+        Frame counters allow you to tell why the frame rate experienced is
+        different from the expected one - for instance, the camera may be
+        slowing down frame production because it's dark, the settings applied
+        may produce decimation of frames, or the system may be CPU starved and
+        therefore dropping frames.
+      </p>
+    </div>
+    <div>
+      <p>The WebIDL changes to add frame counters are the following:
+      <pre class="idl"
+>partial interface MediaStreamTrack {
+  readonly attribute unsigned long long framesEmitted;
+  readonly attribute unsigned long long framesDropped;
+};</pre>
+    </div>
+    <section>
+      <h4>Attributes</h4>
+      <dl data-link-for="MediaStreamTrack" data-dfn-for="MediaStreamTrack"
+      class="attributes">
+        <dt>
+          <dfn>framesEmitted</dfn> of type {{unsigned long long}}, readonly
+        </dt>
+        <dd>
+          <p>
+            Only defined for video tracks. The total number of frames emitted by
+            the track's source for consumption by one or more sinks. See also
+            {{framesDropped}}.
+          </p>
+        </dd>
+        <dt>
+          <dfn>framesDropped</dfn> of type {{unsigned long long}}, readonly
```

Comment:
We could expose capturedFrames instead of framesDropped, it's easy to calculate one from the other.

I'm interested in the delta between capturedFrames and framesEmitted as a signal that frames are getting lost somewhere. Other than frame decimation, it could be a sign of CPU overload